roles/jupyterhub/server/templates/jupyterhub_start.sh.j2

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-{{ jupyterhub_install_dir }}/bin/python -m jupyterhub.app -f {{ jupyterhub_server_conf_dir }}/jupyterhub_config.py &>> {{ jupyterhub_log_dir }}/jupyterhub.log
+{{ jupyterhub_install_dir }}/bin/python -m jupyterhub.app -f {{ jupyterhub_server_conf_dir }}/jupyterhub_config.py {% if jupyterhub_root_logger != "RFA" and jupyterhub_root_logger != "DRFA"%}&>> {{ jupyterhub_log_dir }}/{{ jupyterhub_log_file }}{% endif %}
```

Comment:
Space character missing at the end of the condition ```{% if condition %}```.

Reply:
thanks !

Reply:
done